### ISSUE TITLE
the context menu items not becomes in focus and the context menu title doesn't announce until we press arrow keys haven't been resolved yet, but it should've been fixed in qt 6.4 rc, but we are already in qt 6.5 in teamtalk v5.13, read the discription

### ISSUE THREAD

**Issue by amirmahdifard** [BearWare/TeamTalk5#1925]
### Description

@CoBC remember the time i reported a bug about when i pressed application on the files list to open the context menu, nvda didn't announce anything until i press one of my arrows so then it would announce the title and the first item in focus, you gave me a qt bug report link and when i looked at that, i figgered out that qt developers have fixed that bug in qt 6.4 rc, but i beleaved that teamtalk is now using that version of qt or even abuv it by looking at the about dialog in teamtalk, but that bug with the context menu haven't been fixed and i still have to press my arrow key to get the title and first child item announced: do you know any idea about this, or in faqt are you able to resolv this somehow? thank you in advance, i'll be waiting for your responce.

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by amirmahdifard** [comment]
@CoBC here's your comment related this issue
https://github.com/BearWare/TeamTalk5/issues/1559#issuecomment-1297333317
hope you see this: thank you.

**Comment by CoBC** [comment]
Not fixed for context menus yet. Qt devs should fix more issues this month.

**Comment by amirmahdifard** [comment]
@CoBC ok then, so please close this issue when that is fixed.
